src/components/PageLayout/PageLayout.tsx

```diff
@@ -16,31 +18,46 @@ type PageLayoutProps = {
   children: ReactNode
 }
 
+export type Seminar = 'strom' | 'matik' | 'malynar'
+const seminarTitle: Record<Seminar, string> = {
+  strom: 'STROM',
+  matik: 'Matik',
+  malynar: 'Malynár',
+}
+
 // pre pouzitie len na seminarovych strankach a podstrankach - `/matik(/*)`
 // ked budeme potrebovat top-level stranky ako `/ina-stranka`, budeme musiet upravit, ako sa pracuje s `useSeminarInfo`
 export const PageLayout: FC<PageLayoutProps> = ({contentWidth = 2, title = '', children}) => {
+  const {seminar} = useSeminarInfo()
   return (
-    <PageTitleContainer.Provider initialState={title}>
-      <BannerContainer.Provider>
-        <div className={styles.pageContainer}>
-          <TopGrid />
-          <MenuMain />
-          <div className={styles.grid}>
-            <StromLogo />
-            <div
-              className={clsx(
-                styles.mainContent,
-                contentWidth === 1 && styles.col1,
-                contentWidth === 2 && styles.col2,
-                contentWidth === 3 && styles.col3,
-              )}
-            >
-              {children}
+    <>
+      <Head>
+        <title>
+          {title} | {seminarTitle[seminar]}
```

Comment:
title je v PageLayout optional, mozno na niektorych strankach naozaj nie je, no a nechceli by sme, aby title zacinal ciarou.
tento riadok by som zmenil na... :thinking: prvy nastrel:
```ts
{title && `${title} | `}{seminarTitle[seminar]}
```
druhy nastrel:
```ts
{title ? `${title} | ${seminarTitle[seminar]}` : seminarTitle[seminar]}
```
mozes si vybrat 😄 v druhom pripade by sa dala extrahovat nejaka konstanta pre `seminarTitle[seminar]`, nech je to kratsie.